app/helpers/dttp_mappings_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module DttpMappingsHelper
```

Comment:
I don't think this is really 'a helper'. I wonder if we should have a services concerns directory with this sort of stuff in to follow the convention from models and controllers. I think we've stuck other stuff like `ServicePattern` in lib which isn't right either. Maybe call it `HasDttpMapping` or something 🤷

Reply:
Yeah I struggled where to put this - I started making something like this: https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/lib/dttp/mappable.rb but as a `Dttp::ReverseMappable` directory but then I saw the conflicts is was going to cause and stopped..

Reply:
Agreed it's not a helper, happy to make a services concerns directory 👍